From: Chow Loong Jin <hyperair@debian.org>
Date: Mon, 27 Apr 2026 21:12:52 +0800
Subject: Move hyprland.pc into libdir

The headers are arch-specific (contain library ABI versions), so hyprland.pc
which encodes the path to the arch-specific headers also needs to live in an
arch-specific path.

Forwarded: not-needed
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3c525f6..4fcc159 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -679,7 +679,7 @@ install(FILES ${MANPAGES} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 
 # pkgconfig entry
 install(FILES ${CMAKE_BINARY_DIR}/hyprland.pc
-        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/pkgconfig)
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 
 # protocol headers
 set(HEADERS_PROTO "${CMAKE_CURRENT_SOURCE_DIR}/protocols")
